.github/actions/setup-spark-builder/action.yaml

```diff
@@ -19,13 +19,11 @@ name: Setup Spark Builder
 description: 'Setup Apache Spark to run SQL tests'
 inputs:
   spark-short-version:
-    description: 'The Apache Spark short version (e.g., 3.4) to build'
+    description: 'The Apache Spark short version (e.g., 3.5) to build'
     required: true
-    default: '3.4'
```

Comment:
I removed the defaults since these are required parameters, and we always specify them.